src/NServiceBus.Core.Tests/Persistence/InMemory/When_completing_a_saga_with_no_defined_unique_property.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.Core.Tests.Persistence.InMemory
+{
+    using System;
+    using System.Threading.Tasks;
+    using Extensibility;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_completing_a_saga_with_no_defined_unique_property
```

Comment:
You don't need to have a correlation property configured when you are using a custom finder. This is what led down the rabbit hole of creating these tests.